src/components/actions/helpers.ts

```diff
@@ -44,7 +46,52 @@ export const swapOutAliasedActions = async (templateId: number, action: ActionIn
   if (altCondition !== undefined) aliasedAction.condition = altCondition
 
   // Override parameters
-  aliasedAction.parameter_queries = { ...aliasedAction.parameter_queries, ...overrideParams }
+  aliasedAction.parameter_queries = merge(aliasedAction.parameter_queries, overrideParams)
```

Comment:
Same here.

There's still a problem when you get a mixture of evaluated and pre-evaluated properties in the same object, but that's a pretty edge case. Something to bear in mind though.

Reply:
hmm, are you sure that's desired behaviour, don't we want to override the whole parameter ?

Reply:
I made an issue to consider this specific case (https://github.com/openmsupply/conforma-server/issues/795), but I don't it will cause problems currently, as we're only passing "static" parameters to the preview endpoint / alias action.